bump verify presubmit timeout for old releases

BenTheElder opened the pull request:
This regularly takes 80 minutes, the presubmits are timing out.

https://testgrid.k8s.io/sig-release-1.12-blocking#verify-1.12&graph-metrics=test-duration-minutes

/cc @spiffxp

BenTheElder commented:
xref: https://github.com/kubernetes/kubernetes/pull/77555

spiffxp commented:
/approve
/lgtm
I don't like this one bit, but this is still 15min lower than the timeout for our e2e jobs.

BenTheElder commented:
same on not liking it, it also only applies to older branches though, we've
changed verify a fair bit in the past few releases.

*From: *Kubernetes Prow Robot <notifications@github.com>
*Date: *Tue, May 7, 2019 at 5:22 PM
*To: *kubernetes/test-infra
*Cc: *Benjamin Elder, Mention

@BenTheElder <https://github.com/BenTheElder>: Updated the job-config
> configmap in namespace default using the following files:
>
>    - key generated-security-jobs.yaml using file
>    config/jobs/kubernetes-security/generated-security-jobs.yaml
>    - key verify.yaml using file
>    config/jobs/kubernetes/sig-testing/verify.yaml
>
> In response to this <https://github.com/kubernetes/test-infra/pull/12522>:
>
> This regularly takes 80 minutes, the presubmits are timing out.
>
>
> https://testgrid.k8s.io/sig-release-1.12-blocking#verify-1.12&graph-metrics=test-duration-minutes
>
> /cc @spiffxp <https://github.com/spiffxp>
>
> Instructions for interacting with me using PR comments are available here
> <https://git.k8s.io/community/contributors/guide/pull-requests.md>. If
> you have questions or suggestions related to my behavior, please file an
> issue against the kubernetes/test-infra
> <https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:>
> repository.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/test-infra/pull/12522#issuecomment-490301131>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAHADK5AKUK4RYRUVTGMUEDPUIMNDANCNFSM4HLM7GYA>
> .
>
